apps/svelte.dev/content/docs/kit/60-appendix/40-migrating.md

```diff
@@ -127,9 +127,9 @@ Previously, layout components received a `segment` prop indicating the child seg
 
 ### URLs
 
-In Sapper, all relative URLs were resolved against the base URL — usually `/`, unless the `basepath` option was used — rather than against the current page.
+In Sapper, all relative URLs were resolved against the base URL — usually ` / `, unless the `basepath` option was used — rather than against the current page.
```

Comment:
```suggestion
In Sapper, all relative URLs were resolved against the base URL — usually `/`, unless the `basepath` option was used — rather than against the current page.
```

Overzealous find-replace.